src/types/modules/react-navigation.d.ts

```diff
@@ -0,0 +1,8 @@
+import {RootStackParamList} from '@libs/Navigation/types';
+
+declare global {
+    namespace ReactNavigation {
+        // eslint-disable-next-line
+        interface RootParamList extends RootStackParamList {}
```

Comment:
Why is it needed?

Reply:
https://reactnavigation.org/docs/typescript/#organizing-types
☝️ 

> Specifying a global type for your root navigator would avoid manual annotations in many places.